modules/monitoring-configuring-alert-routing-default-platform-alerts.adoc

```diff
@@ -54,54 +50,87 @@ route:
     - "alertname=Watchdog"
     repeat_interval: 2m
     receiver: watchdog
-  - matchers:
-    - "service=<your_service>" # <5>
-    routes:
-    - matchers:
-      - <your_matching_rules> # <6>
-      receiver: <receiver> # <7>
 receivers:
 - name: default
 - name: watchdog
-- name: <receiver>
-  <receiver_configuration> # <8>
 ----
 <1> If you configured an HTTP cluster-wide proxy, set the `proxy_from_environment` parameter to `true` to enable proxying for all alert receivers.
 <2> Specify how long Alertmanager waits while collecting initial alerts for a group of alerts before sending a notification.
 <3> Specify how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
 <4> Specify the minimum amount of time that must pass before an alert notification is repeated.
 If you want a notification to repeat at each group interval, set the `repeat_interval` value to less than the `group_interval` value.
 The repeated notification can still be delayed, for example, when certain Alertmanager pods are restarted or rescheduled.
-<5> Specify the name of the service that fires the alerts.
-<6> Specify labels to match your alerts.
-<7> Specify the name of the receiver to use for the alerts.
-<8> Specify the receiver configuration.
+
+.. Add your alert receiver configuration:
 +
-[IMPORTANT]
-====
-* Use the `matchers` key name to indicate the matchers that an alert has to fulfill to match the node.
-Do not use the `match` or `match_re` key names, which are both deprecated and planned for removal in a future release.
+[source,yaml]
+----
+# ...
+receivers:
+- name: default
+- name: watchdog
+- name: <receiver> # <1>
+  <receiver_configuration> # <2>
+# ...
+----
+<1> The receiver name.
+<2> The receiver configuration. The supported receivers are PagerDuty, webhook, email, Slack, and Microsoft Teams.
++
+.Example of configuring PagerDuty as an alert receiver
+[source,yaml]
+----
+# ...
+receivers:
+- name: default
+- name: watchdog
+- name: team-frontend-page
+  pagerduty_configs:
+  - routing_key: ABCD01234EFGHIJ56789
+    http_config: # <1> 
+      proxy_from_environment: true
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/91565/files#r2063596008, `proxy_from_environment` is since 4.18